swagger.yaml

```diff
@@ -5096,6 +5096,51 @@ definitions:
       ntp_source:
         type: string
         x-omitempty: false
+      cluster_networks_ipv4:
+        type: array
+        items:
+          type: object
+          $ref: '#/definitions/cluster_network'
+      cluster_networks_ipv6:
+        type: array
+        items:
+          type: object
+          $ref: '#/definitions/cluster_network'
+      cluster_networks_dualstack:
+        type: array
+        items:
+          type: object
+          $ref: '#/definitions/cluster_network'
+      service_networks_ipv4:
+        type: array
+        items:
+          type: object
+          $ref: '#/definitions/service_network'
+      service_networks_ipv6:
+        type: array
+        items:
+          type: object
+          $ref: '#/definitions/service_network'
+      service_networks_dualstack:
+        type: array
+        items:
+          type: object
+          $ref: '#/definitions/service_network'
+      machine_networks_ipv4:
+        type: array
+        items:
+          type: object
+          $ref: '#/definitions/machine_network'
+      machine_networks_ipv6:
```

Comment:
out of curiosity, what is the use of machine_networks default value? the UI doesn't use it.

Reply:
Please remove anything that isn't needed

Reply:
@batzionb for dual-stack in the UI the user *has to* provide machine networks. My understanding was that you need IPv6 defaults to build UI for dual-stack. If you don't need Machine Networks defaults, how is the UI going to work?

Reply:
Machine networks is a select field, the UI provides options based on the host_networks field that comes from the backend and the UI updates the machine_networks field with the user's selection.
The fields cluster_network_cidr, cluster_network_host_prefix and service_network_cidr the UI initially populates from the default-config API.

Reply:
Ok, thanks. Let's make sure we have extensive test for that to avoid problems. We have lately seen some strange behaviour around machine CIDRs and the UI (I think we got those covered already if I track correctly), so we should try testing dual-stack as early in the process as possible. Just let me know once you need some assistance.

For here - yes, I'll remove all the defaults relating to machine networks then

Reply:
So turns out the cloud doesn't even support IPv6 without dual stack, so we only need defaults for ipv4 and dual stack. the ipv6 defaults don't hurt but if you don't want to send fields that won't be used immediately, you can remove them